### DISCUSSION TITLE
swap-on-zram by default on MocaccinoOS desktop

### DISCUSSION THREAD

**Discussion by yusssufff** [mocaccinoOS/mocaccino#61] [General] [answered]
There is a shift towards default " swap on zram " on many serious linux distributions these days: Fedora, EndlessOs, Calculate-Linux, AltLinux...
There are various benefits for the company or home user.
Mostly the system is much more reactive whenever starting to swap (swap on zram being much more reactive than swap on hdd or ssd).

What about joining the move, and enabling swap on zram " by default " in MocaccinoOS Desktop, like Calculate, Endless, Fedora, Alt...?
It would be very helpful for the typical case of home users who have small RAM amounts available and experience regular swapping.

**Comment by mudler** [comment]
Hi, some work towards this has already started. At the moment we don't ship it by default, but you should be good to go to enable it by installing the zram profile and reboot:

`luet install system-profile/zram`

**Comment by mudler** [comment, marked as answer]
Closing the discussion, an issue is already present to track this: https://github.com/mocaccinoOS/mocaccino/issues/64
